ironfish-cli/src/commands/wallet/which.ts

```diff
@@ -33,10 +33,10 @@ export class WhichCommand extends IronfishCommand {
 
     if (!accountName) {
       this.log(
-        'There is currently no account being used.\n' +
-          ' * Create an account: "ironfish accounts:create"\n' +
-          ' * List all accounts: "ironfish accounts:list"\n' +
-          ' * Use an existing account: "ironfish accounts:use <name>"',
+        'There is currently no wallet being used.\n' +
+          ' * Create an wallet: "ironfish wallet:create"\n' +
+          ' * List all wallets: "ironfish wallet:list"\n' +
+          ' * Use an existing wallet: "ironfish wallet:use <name>"',
```

Comment:
The changes to the commands here are correct (`wallet:<command>`), but the descriptions should still use `account` (e.g., `Create an account`)